pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -319,7 +323,7 @@ private String getDynamicConfigurationFromStore(String path, String settingName,
     private double getDynamicConfigurationDouble(String path, String settingName, double defaultValue) {
         double result = defaultValue;
         try {
-            String setting = this.getDynamicConfigurationFromStore(path, settingName, null);
+            String setting = this.getDynamicConfigurationFromStore(path, settingName, String.valueOf(defaultValue));
```

Comment:
what is the impact of this change ?
we should add unit tests around this method and `getDynamicConfigurationBoolean`

Reply:
Ok, I will split this patch. And pull a new request for test getDynamicConfigurationBoolean.
Thanks for reviewing .